### ISSUE TITLE
Issues registering v12.1.1 Suite

### ISSUE THREAD

**Issue by todorpr** [rufoa/ableton#5]
First thanks for your efforts creating and maintaining this script @rufoa and @drmext !

Now my pain - when I generate an Authorize.auz file and drag-n-drop it inside Ableton, I get this message:

<img width="444" alt="image" src="https://github.com/user-attachments/assets/12a7fddd-5ddb-48a1-a060-5fc1e3bb9cfe">

When visiting the link, Ableton tells me that I have enough unlocks left, so this not an issue I guess.

When I drag-n-drop my original Authorize.auz downloaded from Ableton consisting only Lite version serial it shows this message: 

<img width="445" alt="image" src="https://github.com/user-attachments/assets/bfa9ab21-316c-4961-bf17-a4e76e43c25a">

What can be wrong as I see in another [issue](https://github.com/rufoa/ableton/issues/4) that you show the same version successfully authorized?

**Comment by rufoa** [comment]
Hi Todor, did you patch `/Applications/Ableton Live 12 Suite.app/Contents/MacOS/Live` (or `Live.exe` on windows) beforehand? This step is required for the auz to work

**Comment by todorpr** [comment]
No, @rufoa. It was not mentioned in the readme file. How can I patch `/Applications/Ableton Live 12 Suite.app/Contents/MacOS/Live` (Mac user)?

Edit: I found another repo that forked yours https://github.com/devilAPI/abletonCracker and iterated on your script adding a patcher. I patched the Live binary with the patcher, however whn I start Ableton now it crashes. In the readme it says it works on Windows and Linux, and I wonder why it doesn't work on Mac.
